app/views/devise/shared/_profile_fields.html.haml

```diff
@@ -83,6 +83,15 @@
     .help-text Required
 
 
+.grid-x.grid-padding-x.grid-margin-y
```

Comment:
Based on the ticket, do we believe users should have the ability to update their own status? I believe admins & volunteer coordinators should be able to update a volunteer's status, but not the user themselves.

Reply:
That's a good point. I added in a check for logged in admin/coordinators to update this. I also added columns on the users dashboard view that display the status and an edit button.